samples/Gallery/Pages/ClipboardPage.fs

```diff
@@ -153,6 +155,7 @@ module ClipboardPage =
             return Cleared
         }
 
+    //TODO Why do this?
```

Comment:
Strictly speaking this is only useful if you want to unit tests the commands, if not you can just use

```fsharp
Program.statefulWithCmd init update
```
Where the `init` and `update` functions will need to return `Cmd.none` or similars

Reply:
I see, thanks for the explanation! That should probably be explained in detail in a prominent place, since this is all over the Gallery source code.